app/services/bing/search_service.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Bing
+  class SearchService
+    USER_AGENT = 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) '\
+                 'AppleWebKit/537.36 (KHTML, like Gecko) Chrome/111.0.0.0 Safari/537.36'
+    BASE_SEARCH_URL = 'https://www.bing.com/search'
+    HEADERS = { 'User-Agent' => USER_AGENT }.freeze
```

Comment:
Is there a reason why we use a different chrome version as user agent?

If yes:
- let's move `headers` into the `FetchWebpageService` as it is only specific to this class, not to the `SearchService` classes.

If no:
- let's move both `headers` and `user_agent` into the `FetchWebpageService` in order to dry the code and simplify the tests ;-)